server.go

```diff
@@ -31,6 +32,25 @@ type User interface {
 	CreateMessageLimit() *uint32
 }
 
+// A mailbox that supports retrieving per-mailbox message size limits.
+type Mailbox interface {
+	// Get the fixed maximum message size in octets that the backend will accept
+	// when creating a new message. If there is no limit, return nil.
+	//
+	// This overrides the global and per-user limit.
+	CreateMessageLimit() *uint32
+}
+
+// StatusSetAppendLimit sets limit value in MailboxStatus object,
+// nil pointer value will remove limit.
+func StatusSetAppendLimit(status *imap.MailboxStatus, value *uint32) {
+	if value != nil {
+		status.Items[StatusAppendLimit] = strconv.FormatUint(uint64(*value), 10)
```

Comment:
You should be able to just put the `uint32` in the `Items`.

Reply:
Fixed in latest commit.